Automattic/wp-calypso
# A4A: Add missing page view events for A4A portal.

This PR adds missing page view tracking events for the A4A portal.

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/455

## Proposed Changes

* Add Tracking events for the following pages
   | Page |
   |--------|
   | `/overview` |
   | `/sites/:category/:siteUrl/:feature`  |
   | `/sites/need-setup` |
   | `/marketplace/products` |
   | `/marketplace/hosting` |
   | `/marketplace/hosting/wpcom` |
   | `/marketplace/hosting/pressable` |
   | `/marketplace/checkout` |
   | `/marketplace/assign-license` |
   | `/marketplace/download-products` |
   | `/referrals` |
   | `/referrals/commissions` |
   | `/signup` | 

## Testing Instructions

* Use the A4A live link and test all pages specified above.
* Confirm that the page track view event is triggered.

### Confirming if the event is fired
* In your browser, use the dev tool to inspect network requests
* Filter all requests for http://pixel.wp.com/t.gif
* Look for `calypso_page_view` events. **Payload's _en** property should say this.

## Pre-merge Checklist
